docker-compose.prod.yml

```diff
@@ -0,0 +1,9 @@
+version: '2'
+
+services:
+  firefly-app:
+    container_name: firefly-iii
```

Comment:
Oh, right! You can do `docker-compose exec firefly-app php artisan <command>` and that will run it inside the app container without needing to know the name.